src/importer.ts

```diff
@@ -52,7 +54,11 @@ export class Importer {
     private buildConsumers(amqpChannel): Observable<ImportingEntity>[] {
         if (!this.config.onlyType) {
             logger.info("The import will consume messages of all entity types");
-            return [this.buildEventConsumer(amqpChannel), this.buildContactConsumer(amqpChannel)];
+            return [
+                this.buildEventConsumer(amqpChannel),
+                this.buildContactConsumer(amqpChannel),
+                this.buildAddressBookConsumer(amqpChannel),
+            ];
         }
 
         switch (this.config.onlyType) {
```

Comment:
replacing onlyType? with featuresList: BatchOperationType[] would make sense and allow to selectively import any combination from event, contact and addressbook.
the CLI parser would have to be able to handle multiple entries switches, though (eg --features EVENT, CONTACT, ADDRESS) 
WDYT ?